src/platforms/python/guides/fastapi/index.mdx

```diff
@@ -80,7 +68,11 @@ Visiting `"/sentry-debug"` will trigger an error that will be captured by Sentry
 
 ## Options
 
-You can pass the following keyword arguments to `FastApiIntegration()`:
+If you want to change the default behavior of the FastAPI integration, you need to instantiate the integration by hand and pass it to Sentry's `init` function.
+
+Because FastAPI is based on the Starlette framework both integrations `StarletteIntegration`, and `FastApiIntegration` must be instantiated.
+
+You can pass the following keyword arguments to `StarletteIntegration()` respectively `FastApiIntegration()`:
```

Comment:
I'm assuming this was copied from somewhere where there were multiple options in the list below so the order of them mattered, in which case, the word "respectively" would make sense. But here it doesn't work because there's only one option, so there's no order to respect. Also, we shouldn't have a bullet list with only one bullet item.

```suggestion
You can pass the keyword argument `transaction_style` to `StarletteIntegration()` and `FastApiIntegration()`.
```